plugins/inputs/nats_consumer/README.md

```diff
@@ -17,6 +17,9 @@ instances of telegraf can read from a NATS cluster in parallel.
   ## subject(s) to consume
   subjects = ["telegraf"]
 
+  ## jetstream subjects
+  jetstream_subjects = ["js_telegraf"]
```

Comment:
Can you maybe add some more words on what this option does and where we can get an introduction to what `jetstream`s are? I doesn't have to be in the "config" section, but some more explanation will help new users to find their way...

Reply:
I have added some more information to sample.config and README.md